app/controllers/poms_controller.rb

```diff
@@ -10,8 +10,16 @@ def index
     active_poms, inactive_poms = poms.partition { |pom|
       %w[active unavailable].include? pom.status
     }
-    @active_poms = active_poms.map { |p| PomPresenter.new(p) }
-    @inactive_poms = inactive_poms.map { |p| PomPresenter.new(p) }
+
+    if @prison.womens?
+      # poms are split into groups - 1) probation 2) prison & are sorted by last name
+      @active_probation_poms = active_poms.sort_by(&:last_name).select { |pom| pom.position == 'PO' }.map { |po| PomPresenter.new(po) }
```

Comment:
Rather than do this, our pattern so far has been to create a brand-new controller using routing as the switch. It's really confusing having a switch on women's in both controller and view IMHO